minari/dataset/minari_dataset.py

```diff
@@ -162,12 +165,34 @@ def recover_environment(self, eval_env: bool = False, **kwargs) -> gym.Env:
         """Recover the Gymnasium environment used to create the dataset.
 
         Args:
-            eval_env (bool): if True the returned Gymnasium environment will be that intended to be used for evaluation. If no eval_env was specified when creating the dataset, the returned environment will be the same as the one used for creating the dataset. Default False.
+            eval_env (bool): if True, the returned Gymnasium environment will be that intended to be used for evaluation. If no eval_env was specified when creating the dataset, the returned environment will be the same as the one used for creating the dataset. Default False.
+            install_requirements (bool): if True, eventual requirements will be installed before creating the environment. Default False.
```

Comment:
Is the word "eventual" here a typo?

Reply:
Oh, I actually removed this flag, leaving only the warnings, so I remove this line, thanks!